src/Core/src/Core/IFrameworkElement.cs

```diff
@@ -42,6 +43,11 @@ public interface IFrameworkElement
 		/// </summary>
 		IFrameworkElement? Parent { get; }
 
+		/// <summary>
+		/// Gets the shape used to define the outline of the contents of a View.
+		/// </summary>
+		IShape? ClipShape { get; }
```

Comment:
The property name, should be `Clip` or `ClipShape`?

Reply:
Probably clip